tools/Makefile.shared

```diff
@@ -1,4 +1,4 @@
-#**************************************************************************
+#************************-*-makefile-*-************************************
```

Comment:
You can't change the header format without talking to me first. See `tools/check-typo`.